timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/dropwizard/endpoints/RdfUpload.java

```diff
@@ -57,13 +57,20 @@ public Response upload(@FormDataParam("file") final InputStream rdfInputStream,
       return response;
     }
 
+    Optional<MediaType> mediaTypeOpt = getMediaType(body);
```

Comment:
getMediaType(body) is often outside of the control of the user (for example: I don't know how to set the required mime-block-header value properly when using curl)

I think we should move this one to the same approach that we use in the TabularUpload: have the user specify an explicit type FormDataParam